xbmc/guilib/Texture.cpp

```diff
@@ -217,7 +227,7 @@ std::unique_ptr<CTexture> CTexture::LoadFromFile(const std::string& texturePath,
     }
   }
 #endif
-  std::unique_ptr<CTexture> texture = CTexture::CreateTexture();
+  std::unique_ptr<CTexture> texture = CTexture::CreateTexture(0, 0, format);
```

Comment:
Is there any use for the default params in `CreateTexture` anymore?

Reply:
Should I remove them?